multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -72,6 +72,31 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # Price (Dollars) to bid for spot instances. Omit for on-demand instances.
 # workerSpotPrice: "0.05"
 
+## Etcd Cluster config
+## WARNING: Any changes to etcd parameters after the cluster is first created will not be applied
+## during a cluster upgrade, due to concerns over data loss.
+## This situation is being rectified with work towards automated management of etcd clusters
+
+# Number of etcd nodes
+# (Set to an odd number >= 3 for HA control plane)
+# etcdCount: 1
```

Comment:
Two things:

- There's a discrete etcd cluster and controllers in an ASG, so `etcdCount` and `controllerCount` are entirely decoupled now. TLDR; you can modify controller count, etcdCount is set-on-create then no update

- I forgot to doc it in `cluster.yaml`, but it's exposed by the `Cluster` object. I'll fix that up.